src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/ILspLogger.cs

```diff
@@ -3,8 +3,6 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Threading;
-using System.Threading.Tasks;
```

Comment:
Oh - can you obsolete the old interface as well?  Would like to move everyone over to the new class

Reply:
Obsoleting ILspLogger is too viral to be part of this PR.